lib/utils/file_utils.dart

```diff
@@ -53,6 +55,23 @@ class FileUtils {
     return null;
   }
 
+  // This function will take an image, compress it and returns a file
+  static Future<File?> compressAndGetFile(
```

Comment:
File Utils should not have static method. And since its using two third part packages. It should abstract it out.